core/tests/webdriverio_utils/forms.js

```diff
@@ -282,14 +283,14 @@ var RichTextEditor = async function(elem) {
           'Video', 'Image', 'Collapsible', 'Tabs'
         ].includes(componentName)) {
         // RteElementFirst can be found in beginning of RichTextEditor function.
-        await action.setValue(
+        await action.addValue(
```

Comment:
As setValue clears the element first before adding the text (PTAL [here](https://webdriver.io/docs/api/element/setValue/) ) But we do not need to clear the element here at first so I am using addValue. Also this was not throwing error before, because this code is running only in extension suite which is migrated in this PR.